_episodes/format.md

```diff
@@ -57,190 +60,215 @@ Tabular data have specific properties that are better supported by the spreadshe
 >
 {: .discussion}
 
+> ## FAIR principles
+>
+> Let's have a look at [FAIR
+> principles](https://www.go-fair.org/fair-principles/). According to the
+> principles, how does a file format improve FAIRness of research data?
+>
+>> ## Solution
+>>
+>> - I1:(meta) data use a formal, accessible, shared, and broadly applicable
+>>   language for knowledge representation. For example, if data is provided in
+>>   commonly understood and preferably open formats.
+>>
+>> - R1.3: (meta)data meet domain-relevant community standards e.g. those for
+>   data formats.
+> {: .solution}
+{: .challenge}
```

Comment:
Maybe we can make this consistent with e.g. episode on data access. There it's a checkbox like: "This is what go-FAIR.org has to say about Accessible"

Reply:
Thanks, Done!